.env.default

```diff
@@ -0,0 +1,5 @@
+# Simply copy this info into your .env file and add the values relating to your database
```

Comment:
You may want to name this file `env.default` or some such, because `.env.default`, like `.env`, is a hidden file.

Reply:
Yeah, that is true. I can change.